app/forms/conversion/create_project_form.rb

```diff
@@ -162,12 +162,18 @@ def save
 
   def new_tasks_data
     Conversion::TasksData.new(
-      church_supplemental_agreement_not_applicable: church_supplemental_agreement_not_applicable?
+      church_supplemental_agreement_not_applicable: church_supplemental_agreement_not_applicable?,
+      sponsored_support_grant_not_applicable: sponsored_support_grant_not_applicable?
     )
   end
 
   private def church_supplemental_agreement_not_applicable?
     return true unless establishment.has_diocese?
     false
   end
+
+  private def sponsored_support_grant_not_applicable?
+    return true if directive_academy_order == false
```

Comment:
Could probably shorten this to `!directive_academy_order` but it might be less clear